storybook/src/elements/nuxeo-card/nuxeo-card.stories.js

```diff
@@ -0,0 +1,27 @@
+import { storiesOf } from '@storybook/polymer';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { html } from 'lit-html';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-card';
+
+const ICONS = {
+  attachment: 'nuxeo:attachment',
+  dashboard: 'nuxeo:dashboard',
+  edit: 'nuxeo:edit',
+  none: '',
+};
+
+storiesOf('UI/nuxeo-card', module).add('nuxeo-card', () => {
+  const heading = text('Heading', 'About Nuxeo');
+  const icon = select('Icon', ICONS, ICONS.attachment);
+  const collapsible = boolean('Collapsible', false);
+  const opened = boolean('Opened', false);
+  return html`
+    <nuxeo-card heading="${heading}" icon="${icon}" ?collapsible="${collapsible}" ?opened="${opened}">
```

Comment:
Once we toggle collapsible on and off again, we can no longer see the card content. This might be an issue on the element it self. If this is the case, we should create a ticket to fix it.